The job **`aarch64-gnu-llvm-21-2`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/28250397914/job/83700877399) [(plain enhanced)](https://triage.rust-lang.org/gha-logs/rust-lang/rust/83700877399) [(plain)](https://github.com/rust-lang/rust/commit/00da1cceb9967569fc42eb1684bbdc4847f2edaf/checks/83700877399/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
 Documenting core v0.0.0 (/checkout/library/core)
warning: unresolved link to `ilog2`
    --> library/core/src/num/int_macros.rs:3444:15
     |
3444 |           /// [`ilog2`] can produce results more efficiently for base 2, and [`ilog10`]
     |                 ^^^^^ no item named `ilog2` in scope
     |
    ::: library/core/src/num/mod.rs:376:5
     |
 376 | /     int_impl! {
 377 | |         Self = i8,
---

warning: unresolved link to `ilog10`
    --> library/core/src/num/int_macros.rs:3444:78
     |
3444 |           /// [`ilog2`] can produce results more efficiently for base 2, and [`ilog10`]
     |                                                                                ^^^^^^ no item named `ilog10` in scope
     |
    ::: library/core/src/num/mod.rs:376:5
     |
 376 | /     int_impl! {
 377 | |         Self = i8,
---

warning: unresolved link to `ilog2`
    --> library/core/src/num/int_macros.rs:3444:15
     |
3444 |           /// [`ilog2`] can produce results more efficiently for base 2, and [`ilog10`]
     |                 ^^^^^ no item named `ilog2` in scope
     |
    ::: library/core/src/num/mod.rs:401:5
     |
 401 | /     int_impl! {
 402 | |         Self = i16,
---

warning: unresolved link to `ilog10`
    --> library/core/src/num/int_macros.rs:3444:78
     |
3444 |           /// [`ilog2`] can produce results more efficiently for base 2, and [`ilog10`]
     |                                                                                ^^^^^^ no item named `ilog10` in scope
     |
    ::: library/core/src/num/mod.rs:401:5
     |
 401 | /     int_impl! {
 402 | |         Self = i16,
---

warning: unresolved link to `ilog2`
    --> library/core/src/num/int_macros.rs:3444:15
     |
3444 |           /// [`ilog2`] can produce results more efficiently for base 2, and [`ilog10`]
     |                 ^^^^^ no item named `ilog2` in scope
     |
    ::: library/core/src/num/mod.rs:426:5
     |
 426 | /     int_impl! {
 427 | |         Self = i32,
---

warning: unresolved link to `ilog10`
    --> library/core/src/num/int_macros.rs:3444:78
     |
3444 |           /// [`ilog2`] can produce results more efficiently for base 2, and [`ilog10`]
     |                                                                                ^^^^^^ no item named `ilog10` in scope
     |
    ::: library/core/src/num/mod.rs:426:5
     |
 426 | /     int_impl! {
 427 | |         Self = i32,
---

warning: unresolved link to `ilog2`
    --> library/core/src/num/int_macros.rs:3444:15
     |
3444 |           /// [`ilog2`] can produce results more efficiently for base 2, and [`ilog10`]
     |                 ^^^^^ no item named `ilog2` in scope
     |
    ::: library/core/src/num/mod.rs:451:5
     |
 451 | /     int_impl! {
 452 | |         Self = i64,
---

warning: unresolved link to `ilog10`
    --> library/core/src/num/int_macros.rs:3444:78
     |
3444 |           /// [`ilog2`] can produce results more efficiently for base 2, and [`ilog10`]
     |                                                                                ^^^^^^ no item named `ilog10` in scope
     |
    ::: library/core/src/num/mod.rs:451:5
     |
 451 | /     int_impl! {
 452 | |         Self = i64,
---

warning: unresolved link to `ilog2`
    --> library/core/src/num/int_macros.rs:3444:15
     |
3444 |           /// [`ilog2`] can produce results more efficiently for base 2, and [`ilog10`]
     |                 ^^^^^ no item named `ilog2` in scope
     |
    ::: library/core/src/num/mod.rs:476:5
     |
 476 | /     int_impl! {
 477 | |         Self = i128,
---

warning: unresolved link to `ilog10`
    --> library/core/src/num/int_macros.rs:3444:78
     |
3444 |           /// [`ilog2`] can produce results more efficiently for base 2, and [`ilog10`]
     |                                                                                ^^^^^^ no item named `ilog10` in scope
     |
    ::: library/core/src/num/mod.rs:476:5
     |
 476 | /     int_impl! {
 477 | |         Self = i128,
---

warning: unresolved link to `ilog2`
    --> library/core/src/num/int_macros.rs:3444:15
     |
3444 |           /// [`ilog2`] can produce results more efficiently for base 2, and [`ilog10`]
     |                 ^^^^^ no item named `ilog2` in scope
     |
    ::: library/core/src/num/mod.rs:556:5
     |
 556 | /     int_impl! {
 557 | |         Self = isize,
---

warning: unresolved link to `ilog10`
    --> library/core/src/num/int_macros.rs:3444:78
     |
3444 |           /// [`ilog2`] can produce results more efficiently for base 2, and [`ilog10`]
     |                                                                                ^^^^^^ no item named `ilog10` in scope
     |
    ::: library/core/src/num/mod.rs:556:5
     |
 556 | /     int_impl! {
 557 | |         Self = isize,
---

warning: unresolved link to `ilog2`
    --> library/core/src/num/uint_macros.rs:1724:15
     |
1724 |           /// [`ilog2`] can produce results more efficiently for base 2, and [`ilog10`]
     |                 ^^^^^ no item named `ilog2` in scope
     |
    ::: library/core/src/num/mod.rs:583:5
     |
 583 | /     uint_impl! {
 584 | |         Self = u8,
---

warning: unresolved link to `ilog10`
    --> library/core/src/num/uint_macros.rs:1724:78
     |
1724 |           /// [`ilog2`] can produce results more efficiently for base 2, and [`ilog10`]
     |                                                                                ^^^^^^ no item named `ilog10` in scope
     |
    ::: library/core/src/num/mod.rs:583:5
     |
 583 | /     uint_impl! {
 584 | |         Self = u8,
---

warning: unresolved link to `ilog2`
    --> library/core/src/num/uint_macros.rs:1724:15
     |
1724 |           /// [`ilog2`] can produce results more efficiently for base 2, and [`ilog10`]
     |                 ^^^^^ no item named `ilog2` in scope
     |
    ::: library/core/src/num/mod.rs:1231:5
     |
1231 | /     uint_impl! {
1232 | |         Self = u16,
---

warning: unresolved link to `ilog10`
    --> library/core/src/num/uint_macros.rs:1724:78
     |
1724 |           /// [`ilog2`] can produce results more efficiently for base 2, and [`ilog10`]
     |                                                                                ^^^^^^ no item named `ilog10` in scope
     |
    ::: library/core/src/num/mod.rs:1231:5
     |
1231 | /     uint_impl! {
1232 | |         Self = u16,
---

warning: unresolved link to `ilog2`
    --> library/core/src/num/uint_macros.rs:1724:15
     |
1724 |           /// [`ilog2`] can produce results more efficiently for base 2, and [`ilog10`]
     |                 ^^^^^ no item named `ilog2` in scope
     |
    ::: library/core/src/num/mod.rs:1287:5
     |
1287 | /     uint_impl! {
1288 | |         Self = u32,
---

warning: unresolved link to `ilog10`
    --> library/core/src/num/uint_macros.rs:1724:78
     |
1724 |           /// [`ilog2`] can produce results more efficiently for base 2, and [`ilog10`]
     |                                                                                ^^^^^^ no item named `ilog10` in scope
     |
    ::: library/core/src/num/mod.rs:1287:5
     |
1287 | /     uint_impl! {
1288 | |         Self = u32,
---

warning: unresolved link to `ilog2`
    --> library/core/src/num/uint_macros.rs:1724:15
     |
1724 |           /// [`ilog2`] can produce results more efficiently for base 2, and [`ilog10`]
     |                 ^^^^^ no item named `ilog2` in scope
     |
    ::: library/core/src/num/mod.rs:1319:5
     |
1319 | /     uint_impl! {
1320 | |         Self = u64,
---

warning: unresolved link to `ilog10`
    --> library/core/src/num/uint_macros.rs:1724:78
     |
1724 |           /// [`ilog2`] can produce results more efficiently for base 2, and [`ilog10`]
     |                                                                                ^^^^^^ no item named `ilog10` in scope
     |
    ::: library/core/src/num/mod.rs:1319:5
     |
1319 | /     uint_impl! {
1320 | |         Self = u64,
---

warning: unresolved link to `ilog2`
    --> library/core/src/num/uint_macros.rs:1724:15
     |
1724 |           /// [`ilog2`] can produce results more efficiently for base 2, and [`ilog10`]
     |                 ^^^^^ no item named `ilog2` in scope
     |
    ::: library/core/src/num/mod.rs:1351:5
     |
1351 | /     uint_impl! {
1352 | |         Self = u128,
---

warning: unresolved link to `ilog10`
    --> library/core/src/num/uint_macros.rs:1724:78
     |
1724 |           /// [`ilog2`] can produce results more efficiently for base 2, and [`ilog10`]
     |                                                                                ^^^^^^ no item named `ilog10` in scope
     |
    ::: library/core/src/num/mod.rs:1351:5
     |
1351 | /     uint_impl! {
1352 | |         Self = u128,
---

warning: unresolved link to `ilog2`
    --> library/core/src/num/uint_macros.rs:1724:15
     |
1724 |           /// [`ilog2`] can produce results more efficiently for base 2, and [`ilog10`]
     |                 ^^^^^ no item named `ilog2` in scope
     |
    ::: library/core/src/num/mod.rs:1449:5
     |
1449 | /     uint_impl! {
1450 | |         Self = usize,
---

warning: unresolved link to `ilog10`
    --> library/core/src/num/uint_macros.rs:1724:78
     |
1724 |           /// [`ilog2`] can produce results more efficiently for base 2, and [`ilog10`]
     |                                                                                ^^^^^^ no item named `ilog10` in scope
     |
    ::: library/core/src/num/mod.rs:1449:5
     |
1449 | /     uint_impl! {
1450 | |         Self = usize,
---
error: warnings are denied by `build.warnings` configuration
warning: build failed, waiting for other jobs to finish...
[RUSTC-TIMING] core test:false 26.697
Bootstrap failed while executing `--stage 2 test --skip tidy --skip src/tools/rust-analyzer --skip tests --skip coverage-map --skip coverage-run --skip library --skip tidyselftest`
Command `/checkout/obj/build/aarch64-unknown-linux-gnu/stage0/bin/cargo doc --target aarch64-unknown-linux-gnu -Zbinary-dep-depinfo -j 4 -Zroot-dir=/checkout -Zno-embed-metadata --locked --color=always --profile=dist -p alloc -p compiler_builtins -p core -p panic_abort -p panic_unwind -p proc_macro -p rustc-std-workspace-core -p std -p std_detect -p sysroot -p test -p unwind --features 'backtrace panic-unwind compiler-builtins-c' --manifest-path /checkout/library/sysroot/Cargo.toml --no-deps --target-dir /checkout/obj/build/aarch64-unknown-linux-gnu/stage1-std/aarch64-unknown-linux-gnu/doc -Zskip-rustdoc-fingerprint -Zrustdoc-map [workdir=/checkout]` failed with exit code 101
Created at: src/bootstrap/src/core/build_steps/doc.rs:809:21
Executed at: src/bootstrap/src/core/build_steps/doc.rs:842:22

--- BACKTRACE vvv
   0: std::backtrace_rs::backtrace::libunwind::trace
             at /rustc/0417c25868d6dfbd1c291dfeae950504faa6f790/library/std/src/../../backtrace/src/backtrace/libunwind.rs:117:9
   1: std::backtrace_rs::backtrace::trace_unsynchronized::<<std::backtrace::Backtrace>::create::{closure#0}>
             at /rustc/0417c25868d6dfbd1c291dfeae950504faa6f790/library/std/src/../../backtrace/src/backtrace/mod.rs:66:14
   2: <std::backtrace::Backtrace>::create
             at /rustc/0417c25868d6dfbd1c291dfeae950504faa6f790/library/std/src/backtrace.rs:331:13
   3: <bootstrap::utils::exec::DeferredCommand>::finish_process
             at /checkout/src/bootstrap/src/utils/exec.rs:939:17
   4: <bootstrap::utils::exec::DeferredCommand>::wait_for_output::<&bootstrap::utils::exec::ExecutionContext>
             at /checkout/src/bootstrap/src/utils/exec.rs:831:21
   5: <bootstrap::utils::exec::ExecutionContext>::run
             at /checkout/src/bootstrap/src/utils/exec.rs:741:45
   6: <bootstrap::utils::exec::BootstrapCommand>::run::<&bootstrap::core::builder::Builder>
             at /checkout/src/bootstrap/src/utils/exec.rs:339:27
   7: bootstrap::core::build_steps::doc::doc_std
             at /checkout/src/bootstrap/src/core/build_steps/doc.rs:842:22
   8: <bootstrap::core::build_steps::doc::Std as bootstrap::core::builder::Step>::run
             at /checkout/src/bootstrap/src/core/build_steps/doc.rs:733:9
   9: <bootstrap::core::builder::Builder>::ensure::<bootstrap::core::build_steps::doc::Std>
             at /checkout/src/bootstrap/src/core/builder/mod.rs:1597:36
  10: <bootstrap::core::build_steps::doc::Std as bootstrap::core::builder::Step>::make_run
             at /checkout/src/bootstrap/src/core/build_steps/doc.rs:675:21
  11: <bootstrap::core::builder::StepDescription>::maybe_run
             at /checkout/src/bootstrap/src/core/builder/mod.rs:476:13
  12: bootstrap::core::builder::cli_paths::match_paths_to_steps_and_run
             at /checkout/src/bootstrap/src/core/builder/cli_paths.rs:141:22
  13: <bootstrap::core::builder::Builder>::run_step_descriptions
             at /checkout/src/bootstrap/src/core/builder/mod.rs:1140:9
---
  16: <bootstrap::core::builder::Builder>::ensure::<bootstrap::core::build_steps::test::Linkcheck>
             at /checkout/src/bootstrap/src/core/builder/mod.rs:1597:36
  17: <bootstrap::core::build_steps::test::Linkcheck as bootstrap::core::builder::Step>::make_run
             at /checkout/src/bootstrap/src/core/build_steps/test.rs:140:21
  18: <bootstrap::core::builder::StepDescription>::maybe_run
             at /checkout/src/bootstrap/src/core/builder/mod.rs:476:13
  19: bootstrap::core::builder::cli_paths::match_paths_to_steps_and_run
             at /checkout/src/bootstrap/src/core/builder/cli_paths.rs:141:22
  20: <bootstrap::core::builder::Builder>::run_step_descriptions
             at /checkout/src/bootstrap/src/core/builder/mod.rs:1140:9
  21: <bootstrap::core::builder::Builder>::execute_cli
             at /checkout/src/bootstrap/src/core/builder/mod.rs:1119:14
  22: <bootstrap::Build>::build
             at /checkout/src/bootstrap/src/lib.rs:803:25
  23: bootstrap::main
             at /checkout/src/bootstrap/src/bin/main.rs:130:11
  24: <fn() as core::ops::function::FnOnce<()>>::call_once
             at /rustc/0417c25868d6dfbd1c291dfeae950504faa6f790/library/core/src/ops/function.rs:250:5
  25: std::sys::backtrace::__rust_begin_short_backtrace::<fn(), ()>
             at /rustc/0417c25868d6dfbd1c291dfeae950504faa6f790/library/std/src/sys/backtrace.rs:166:18
  26: std::rt::lang_start::<()>::{closure#0}
             at /rustc/0417c25868d6dfbd1c291dfeae950504faa6f790/library/std/src/rt.rs:206:18
  27: <&dyn core::ops::function::Fn<(), Output = i32> + core::panic::unwind_safe::RefUnwindSafe + core::marker::Sync as core::ops::function::FnOnce<()>>::call_once
             at /rustc/0417c25868d6dfbd1c291dfeae950504faa6f790/library/core/src/ops/function.rs:287:21
  28: std::panicking::catch_unwind::do_call::<&dyn core::ops::function::Fn<(), Output = i32> + core::panic::unwind_safe::RefUnwindSafe + core::marker::Sync, i32>
             at /rustc/0417c25868d6dfbd1c291dfeae950504faa6f790/library/std/src/panicking.rs:581:40
  29: std::panicking::catch_unwind::<i32, &dyn core::ops::function::Fn<(), Output = i32> + core::panic::unwind_safe::RefUnwindSafe + core::marker::Sync>
             at /rustc/0417c25868d6dfbd1c291dfeae950504faa6f790/library/std/src/panicking.rs:544:19
  30: std::panic::catch_unwind::<&dyn core::ops::function::Fn<(), Output = i32> + core::panic::unwind_safe::RefUnwindSafe + core::marker::Sync, i32>
             at /rustc/0417c25868d6dfbd1c291dfeae950504faa6f790/library/std/src/panic.rs:359:14
  31: std::rt::lang_start_internal::{closure#0}
             at /rustc/0417c25868d6dfbd1c291dfeae950504faa6f790/library/std/src/rt.rs:175:24
  32: std::panicking::catch_unwind::do_call::<std::rt::lang_start_internal::{closure#0}, isize>
             at /rustc/0417c25868d6dfbd1c291dfeae950504faa6f790/library/std/src/panicking.rs:581:40
---
  39: __libc_start_main
  40: _start


Command has failed. Rerun with -v to see more details.
Build completed unsuccessfully in 0:29:13
  local time: Fri Jun 26 16:49:46 UTC 2026
  network time: Fri, 26 Jun 2026 16:49:47 GMT
##[error]Process completed with exit code 1.
##[group]Run echo "disk usage:"
```

</details>

--- a/library/core/src/num/int_macros.rs
+++ b/library/core/src/num/int_macros.rs
@@ -983,7 +983,7 @@ macro_rules! int_impl {
         /// This function will always panic on overflow, regardless of whether overflow checks are enabled.
         ///
         /// The only case where such an overflow can occur is when one divides `MIN / -1` on a signed type (where
-        /// `MIN` is the negative minimal value for the type); this is equivalent to `-MIN`, a positive value
+        /// [`MIN`](Self::MIN) is the negative minimal value for the type); this is equivalent to `-MIN`, a positive value
         /// that is too large to represent in the type.
         ///
         /// # Examples
@@ -1050,7 +1050,7 @@ macro_rules! int_impl {
         /// This function will always panic on overflow, regardless of whether overflow checks are enabled.
         ///
         /// The only case where such an overflow can occur is when one divides `MIN / -1` on a signed type (where
-        /// `MIN` is the negative minimal value for the type); this is equivalent to `-MIN`, a positive value
+        /// [`MIN`](Self::MIN) is the negative minimal value for the type); this is equivalent to `-MIN`, a positive value
         /// that is too large to represent in the type.
         ///
         /// # Examples
@@ -1223,7 +1223,7 @@ macro_rules! int_impl {
         /// This function will always panic on overflow, regardless of whether overflow checks are enabled.
         ///
         /// The only case where such an overflow can occur is `x % y` for `MIN / -1` on a
-        /// signed type (where `MIN` is the negative minimal value), which is invalid due to implementation artifacts.
+        /// signed type (where [`MIN`](Self::MIN) is the negative minimal value), which is invalid due to implementation artifacts.
         ///
         /// # Examples
         ///
@@ -1289,7 +1289,7 @@ macro_rules! int_impl {
         /// This function will always panic on overflow, regardless of whether overflow checks are enabled.
         ///
         /// The only case where such an overflow can occur is `x % y` for `MIN / -1` on a
-        /// signed type (where `MIN` is the negative minimal value), which is invalid due to implementation artifacts.
+        /// signed type (where [`MIN`](Self::MIN) is the negative minimal value), which is invalid due to implementation artifacts.
         ///
         /// # Examples
         ///
@@ -2259,8 +2259,8 @@ macro_rules! int_impl {
         /// boundary of the type.
         ///
         /// The only case where such wrapping can occur is when one divides `MIN / -1` on a signed type (where
-        /// `MIN` is the negative minimal value for the type); this is equivalent to `-MIN`, a positive value
-        /// that is too large to represent in the type. In such a case, this function returns `MIN` itself.
+        /// [`MIN`](Self::MIN) is the negative minimal value for the type); this is equivalent to `-MIN`, a positive value
+        /// that is too large to represent in the type. In such a case, this function returns [`MIN`](Self::MIN) itself.
         ///
         /// # Panics
         ///
@@ -2284,9 +2284,9 @@ macro_rules! int_impl {
         /// Wrapping Euclidean division. Computes `self.div_euclid(rhs)`,
         /// wrapping around at the boundary of the type.
         ///
-        /// Wrapping will only occur in `MIN / -1` on a signed type (where `MIN` is the negative minimal value
+        /// Wrapping will only occur in `MIN / -1` on a signed type (where [`MIN`](Self::MIN) is the negative minimal value
         /// for the type). This is equivalent to `-MIN`, a positive value that is too large to represent in the
-        /// type. In this case, this method returns `MIN` itself.
+        /// type. In this case, this method returns [`MIN`](Self::MIN) itself.
         ///
         /// # Panics
         ///
@@ -2311,7 +2311,7 @@ macro_rules! int_impl {
         /// boundary of the type.
         ///
         /// Such wrap-around never actually occurs mathematically; implementation artifacts make `x % y`
-        /// invalid for `MIN / -1` on a signed type (where `MIN` is the negative minimal value). In such a case,
+        /// invalid for `MIN / -1` on a signed type (where [`MIN`](Self::MIN) is the negative minimal value). In such a case,
         /// this function returns `0`.
         ///
         /// # Panics
@@ -2336,8 +2336,8 @@ macro_rules! int_impl {
         /// Wrapping Euclidean remainder. Computes `self.rem_euclid(rhs)`, wrapping around
         /// at the boundary of the type.
         ///
-        /// Wrapping will only occur in `MIN % -1` on a signed type (where `MIN` is the negative minimal value
-        /// for the type). In this case, this method returns 0.
+        /// Wrapping will only occur in `MIN % -1` on a signed type (where [`MIN`](Self::MIN) is
+        /// the negative minimal value for the type). In this case, this method returns 0.
         ///
         /// # Panics
         ///
@@ -2361,9 +2361,9 @@ macro_rules! int_impl {
         /// Wrapping (modular) negation. Computes `-self`, wrapping around at the boundary
         /// of the type.
         ///
-        /// The only case where such wrapping can occur is when one negates `MIN` on a signed type (where `MIN`
+        /// The only case where such wrapping can occur is when one negates [`MIN`](Self::MIN) on a signed type (where [`MIN`](Self::MIN)
         /// is the negative minimal value for the type); this is a positive value that is too large to represent
-        /// in the type. In such a case, this function returns `MIN` itself.
+        /// in the type. In such a case, this function returns [`MIN`](Self::MIN) itself.
         ///
         /// # Examples
         ///
@@ -2460,7 +2460,7 @@ macro_rules! int_impl {
         ///
         /// The only case where such wrapping can occur is when one takes the absolute value of the negative
         /// minimal value for the type; this is a positive value that is too large to represent in the type. In
-        /// such a case, this function returns `MIN` itself.
+        /// such a case, this function returns [`MIN`](Self::MIN) itself.
         ///
         /// # Examples
         ///
@@ -2772,7 +2772,7 @@ macro_rules! int_impl {
         ///
         /// # Examples
         ///
-        /// Please note that this example is shared among integer types, which is why `i32` is used.
+        /// Please note that this example is shared among integer types, which is why [`i32`] is used.
         ///
         /// ```
         /// #![feature(signed_bigint_helpers)]
@@ -2950,7 +2950,7 @@ macro_rules! int_impl {
         /// Negates self, overflowing if this is equal to the minimum value.
         ///
         /// Returns a tuple of the negated version of self along with a boolean indicating whether an overflow
-        /// happened. If `self` is the minimum value (e.g., `i32::MIN` for values of type `i32`), then the
+        /// happened. If `self` is the minimum value (e.g., [`i32::MIN`] for values of type [`i32`]), then the
         /// minimum value will be returned again and `true` will be returned for an overflow happening.
         ///
         /// # Examples
@@ -3020,7 +3020,7 @@ macro_rules! int_impl {
         ///
         /// Returns a tuple of the absolute version of self along with a boolean indicating whether an overflow
         /// happened. If self is the minimum value
-        #[doc = concat!("(e.g., ", stringify!($SelfT), "::MIN for values of type ", stringify!($SelfT), "),")]
+        #[doc = concat!("(e.g., [`", stringify!($SelfT), "::MIN`] for values of type [`", stringify!($SelfT), "`]),")]
         /// then the minimum value will be returned again and true will be returned
         /// for an overflow happening.
         ///
@@ -3177,7 +3177,7 @@ macro_rules! int_impl {
         ///
         /// # Panics
         ///
-        /// This function will panic if `rhs` is zero or if `self` is `Self::MIN`
+        /// This function will panic if `rhs` is zero or if `self` is [`Self::MIN`]
         /// and `rhs` is -1. This behavior is not affected by the `overflow-checks` flag.
         ///
         /// # Examples
@@ -3215,7 +3215,7 @@ macro_rules! int_impl {
         ///
         /// # Panics
         ///
-        /// This function will panic if `rhs` is zero or if `self` is `Self::MIN` and
+        /// This function will panic if `rhs` is zero or if `self` is [`Self::MIN`] and
         /// `rhs` is -1. This behavior is not affected by the `overflow-checks` flag.
         ///
         /// # Examples
@@ -3262,7 +3262,7 @@ macro_rules! int_impl {
         ///
         /// # Panics
         ///
-        /// This function will panic if `rhs` is zero or if `self` is `Self::MIN`
+        /// This function will panic if `rhs` is zero or if `self` is [`Self::MIN`]
         /// and `rhs` is -1. This behavior is not affected by the `overflow-checks` flag.
         ///
         /// # Examples
@@ -3304,7 +3304,7 @@ macro_rules! int_impl {
         ///
         /// # Panics
         ///
-        /// This function will panic if `rhs` is zero or if `self` is `Self::MIN`
+        /// This function will panic if `rhs` is zero or if `self` is [`Self::MIN`]
         /// and `rhs` is -1. This behavior is not affected by the `overflow-checks` flag.
         ///
         /// # Examples
@@ -3441,8 +3441,8 @@ macro_rules! int_impl {
         /// rounded down.
         ///
         /// This method might not be optimized owing to implementation details;
-        /// `ilog2` can produce results more efficiently for base 2, and `ilog10`
-        /// can produce results more efficiently for base 10.
+        /// [`ilog2`][Self::ilog2] can produce results more efficiently for base 2,
+        /// and [`ilog10`](Self::ilog10) can produce results more efficiently for base 10.
         ///
         /// # Panics
         ///

--- a/library/core/src/num/uint_macros.rs
+++ b/library/core/src/num/uint_macros.rs
@@ -1721,8 +1721,8 @@ macro_rules! uint_impl {
         /// rounded down.
         ///
         /// This method might not be optimized owing to implementation details;
-        /// `ilog2` can produce results more efficiently for base 2, and `ilog10`
-        /// can produce results more efficiently for base 10.
+        /// [`ilog2`](Self::ilog2) can produce results more efficiently for base 2,
+        /// and [`ilog10`](Self::ilog10) can produce results more efficiently for base 10.
         ///
         /// # Panics
         ///

